core/src/test/scala/org/apache/spark/util/JsonProtocolSuite.scala

```diff
@@ -378,15 +374,20 @@ class JsonProtocolSuite extends SparkFunSuite {
     val oldInfo = JsonProtocol.accumulableInfoFromJson(oldJson)
     assert(false === oldInfo.internal)
   }
+}
 
-  /** -------------------------- *
-   | Helper test running methods |
-   * --------------------------- */
 
-  private def testEvent(event: SparkListenerEvent, jsonString: String) {
+// This extends SparkFunSuite only because we want its `assert` method.
+private[spark] object JsonProtocolSuite extends SparkFunSuite {
```

Comment:
Also, if you just want `assert` there's a different Scalatest trait that you can mix in without having to have an object extend a suite; I think it's `Matchers` or `Assertions` or something like that.